### DISCUSSION TITLE
Unable to configure Git on PC

### DISCUSSION THREAD

**Discussion by ananya-2023** [Jay-study-nildana/FrontEndForStudents#31] [General]
Good Evng, 

As you have mentioned earlier, even after authenticating /logging in, cloning is failing everytime. could you kindly help: 

```
SMRUTI@SMRUTI-PC MINGW64 ~/Videos/Edureka
$ git clone https://github.com/ananya-2023/practicemarch052023.git
Cloning into 'practicemarch052023'...
fatal: An error occurred while sending the request.
fatal: The request was aborted: Could not create SSL/TLS secure channel.
error: unable to read askpass response from 'C:/Program Files/Git/mingw64/bin/git-askpass.exe'
Username for 'https://github.com': ananya-2023
Smayan!2016
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ananya-2023/practicemarch052023.git/'
```

```
SMRUTI@SMRUTI-PC MINGW64 ~/Videos/Edureka
$ Smayan!2016
bash: !2016: event not found

SMRUTI@SMRUTI-PC MINGW64 ~/Videos/Edureka
$ git clone https://github.com/ananya-2023/practicemar2023.git
Cloning into 'practicemar2023'...
fatal: unable to access 'https://github.com/ananya-2023/practicemar2023.git/': Failed to connect to github.com port 443 after 21676 ms: Couldn't connect to server

SMRUTI@SMRUTI-PC MINGW64 ~/Videos/Edureka
$ git clone https://github.com/ananya-2023/practicemar2023.git
Cloning into 'practicemar2023'...
fatal: An error occurred while sending the request.
fatal: The request was aborted: Could not create SSL/TLS secure channel.
```

unable to troubleshoot or move forward.

**Comment by Jay-study-nildana** [comment]
@ananya-2023 

hello and welcome to my discussion forum. glad to have you here. 

First thing first, you see, as you see in the error, password authentication is no longer allowed on github. I have made a few videos about this. hold on, let me share the link with you.

[Video - Setting Up GitHub VS Code and FireFox](https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/15)

Check that link, where I have put the latest and all the videos related to git. Please remember that it will take up to 2 hours the first time you configure git. So, please be patient and watch the video.

Now, another option is that you can simply use GitHub Desktop. GitHub Desktop is a little more easy to use, so, you can start with GitHub Desktop, slowly, after some time, learn how to use git bash ;) 

Any more questions, please post it here or create new discussion. And of course, follow me on socials (you can find all my socials on my github profile) and stay in touch.

Happy Learning.

Jay

**Reply by ananya-2023** [reply]
Thank You sir. Will so as mentioned. Have a Nice Day Sir.

**Reply by Jay-study-nildana** [reply]
@ananya-2023 

you are most welcome.

Jay.
